templates/types/multiagent/express/examples/workflow.ts

```diff
@@ -0,0 +1,35 @@
+import { StepFunction, Workflow } from "@llamaindex/core/workflow";
+import { ChatMessage } from "llamaindex";
+import { FunctionCallingAgent } from "../agents/single";
+import { createResearcher } from "./researcher";
+
+// TODO: implement
+class BlogPostWorkflow extends Workflow {}
```

Comment:
Good idea, just implement the explicit workflow for TS for now. No orchestrator and no choreography

Reply:
@thucpn but don't subclass `Workflow` for the blog post workflow, just create an instance in `createWorkflow` and call `addStep` on it to add the methods (like in all the LITS workflow examples)

Reply:
each step method will then use the agent directly, e.g. `writer.run`